Service rate considerations (#27)
drvinceknight/upgraded-octo-barnacle: tex/main.tex

\begin{highlights}
\item Mathematical modelling for maximising survival with heterogeneous
      patients and fleets
\item Population based neighbourhood search algorithm for finding better fleet
      allocations

> \item Fixed point numerical solution to equation used to approximate utilisation

Reviewer: ```suggestion
\item Fixed point numerical solution of an equation used to approximate utilisation
```

\tilde{\pi}_a &= \frac{\tilde{\lambda_a}}{\tilde{\mu}} \label{eqn:utilisation_ratio_secondary}
\end{align}

Note that here we assume that average job times are not dependant on the vehicle
location $a$, although this may be unrealistic, given that the travel between
locations are a key part of an vehicle's job time ($B_{pa}$, $D_{ya}$, $F_{pa}$).
However, as discussed in Section~\ref{sec:problem_description}, there are a
number of other components to an ambulance job including time on site ($G_k$),
hospital handover time ($J_{yk}$), and vehicle refill time ($\Theta$). For the
case of Jakarta, we justify the assumption of job times not being location
dependant by considering the proportion of an ambulance job's time that is
location dependant, $\rho$, defined in
Equation~\ref{eqn:proportion_location_dependant},

\begin{equation}\label{eqn:proportion_location_dependant}
\rho = \begin{cases}
  \frac{B_{pa} + D_{ya}}{T_{paky}} & \text{ if hospital required,} \\[8pt]
  \frac{B_{pa} + F_{pa}}{T_{pak}} & \text{ if no hospital required,}
\end{cases}
\end{equation}

\noindent

> then, using the simulation described later in Section~\ref{sec:simulation}, we

Reviewer: ```suggestion
then, using the simulation described in Section~\ref{sec:simulation}, we
```

consider the distribution of $\rho$ across all simulated jobs under the current
situation and allocation in Jakarta, shown in Figure~\ref{fig:rho_distribution}.
We see for primary vehicle jobs on average 24\% of a job time is location

> dependant, while it is 36.5\% for secondary vehicles. Therefore this assumption
> should not have too much of an effect for primary vehicles, whose job times are
> much longer than secondary vehicles, but leaves open questions about how to
> better handle this assumption for secondary vehicles.

Reviewer: This is tricky as I'd argue that 24% is still high.

I suggest the following:

```suggestion
dependant, while it is 36.5\% for secondary vehicles.  Considering heterogenous service rates would be an interesting further research question.
```
